src/community_advisors/models/mod.rs

```diff
@@ -5,15 +5,16 @@ use serde::{Deserialize, Deserializer};
 pub use de::{AdvisorReviewRow, ReviewScore};
 
 pub enum ProposalStatus {
-    Approved,
-    NotApproved,
+    Funded,
```

Comment:
mmm, `Funded` and `Approved` are not synonyms, what's the rationale behind this?

Reply:
This was a missuse. In the original document the strings `Funded` and `NotFunded` are used, so I wanted to reflect the same here. Logically speaking, all proposals that we use in the document are **approved** but not all are **funded**.

Reply:
Can you point me to the document? In the rewards script I'm quite sure those were supposed to be approved ones and not funded

Reply:
You can check the tags [here](https://github.com/input-output-hk/catalyst-toolbox/blob/54ced5daadf6405ef80086c49832a447057e0f3b/scripts/python/proposers_rewards.py#L21)

Reply:
@Zeegomo , You were right, we should be reading from the `meets_approval_threshold` column in the csv. It is fixed now.